controllers/authorino_controller.go

```diff
@@ -196,7 +196,7 @@ func (r *AuthorinoReconciler) buildAuthorinoDeployment(authorino *api.Authorino)
 	var saName = authorino.Name + "-authorino"
 
 	if authorino.Spec.Image == "" {
-		authorino.Spec.Image = fmt.Sprintf("quay.io/3scale/authorino:%s", api.AuthorinoVersion)
+		authorino.Spec.Image = fmt.Sprintf("quay.io/kuadrant/authorino:%s", api.AuthorinoVersion)
```

Comment:
As we are making changes here, it would be good to move it into a const

Reply:
Done